### PR TITLE
add check on null instead of "" for boolean

### PR DESCRIPTION
**Pull request by TimurGit**

### CODE REVIEW

**Comment by TimurGit** [comment]
@exodusanto I use laravel 5.4.8 with default settings. It's contain \Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class. Because $this->userOptions['value'] return null.

File: src/Frozennode/Administrator/Fields/Boolean.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ public function setFilter($filter)
 	public function filterQuery(QueryBuilder &$query, &$selects = null)
 	{
 		//if the field isn't empty
-		if ($this->getOption('value') !== '')
+		if (!is_null($this->getOption('value')))
```
Comment (exodusanto) [inline src/Frozennode/Administrator/Fields/Boolean.php]: @TimurGit Ok, but can you use this syntax ?

```
$value = $this->getOption('value');
if (!is_null($value) && $value !== "")
```

**Comment by TimurGit** [comment]
@exodusanto Thanks, good idea!

File: src/Frozennode/Administrator/Fields/Boolean.php [hunk 2]
```diff
@@ -55,7 +55,8 @@ public function setFilter($filter)
 		$this->userOptions['value'] = $this->validator->arrayGet($filter, 'value', '');
 
 		//if it isn't null, we have to check the 'true'/'false' string
-		if (!is_null($this->userOptions['value']))
```
Comment (exodusanto) [inline src/Frozennode/Administrator/Fields/Boolean.php]: @TimurGit ops... here you have to update
```
$this->getOption('value')
``` 

with 

```
$this->userOptions['value']
```

**Pull request merged**